pyccel/codegen/wrapper/cuda_to_c_wrapper.py

```diff
@@ -0,0 +1,82 @@
+# coding: utf-8
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+"""
+Module describing the code-wrapping class : CudaToPythonWrapper
+which creates an interface exposing Cuda code to C.
+"""
+
+from pyccel.ast.bind_c      import BindCModule
+from pyccel.errors.errors   import Errors
+from pyccel.ast.bind_c      import BindCVariable
+from .wrapper               import Wrapper
+
+errors = Errors()
+
+class CudaToCWrapper(Wrapper):
+    """
+    Class for creating a wrapper exposing Fortran code to C.
+
+    A class which provides all necessary functions for wrapping different AST
+    objects such that the resulting AST is C-compatible. This new AST is
+    printed as an intermediary layer.
+    """
+    def __init__(self):
+        self._wrapper_names_dict = {}
+        super().__init__()
+
+    def _wrap_Module(self, expr):
+        """
+        Create a Module which is compatible with C.
+
+        Create a Module which provides an interface between C and the
+        Module described by expr
+
+        Parameters
+        ----------
+        expr : pyccel.ast.core.Module
+            The module to be wrapped.
+
+        Returns
+        -------
+        pyccel.ast.core.BindCModule
+            The C-compatible module.
+        """
+        funcs = [f for f in expr.funcs if f.is_semantic and not f.is_inline]
+        if expr.init_func:
+            init_func = funcs[next(i for i,f in enumerate(funcs) if f == expr.init_func)]
+        else:
+            init_func = None
```

Comment:
I don't understand why this code is necessary. Can't you just do: `init_func = expr.init_func`?

Reply:
fixed 3580b26 u right